schema/mysql/v57/cadence/schema.sql

```diff
@@ -162,18 +161,20 @@ CREATE TABLE tasks (
   task_type TINYINT NOT NULL, -- {Activity, Decision}
   task_id BIGINT NOT NULL,
   expiry_ts DATETIME(6) NOT NULL,
-  PRIMARY KEY (shard_id, domain_id, task_list_name, task_type, task_id)
+  PRIMARY KEY (domain_id, task_list_name, task_type, task_id)
```

Comment:
should we just create a tasklist id for each tasklist, instead of using domain, name and type as key.  Specially considering task_list_name is user defined string and could be large.

Reply:
that's a good idea - then the mapping from tasklistID -> taskListName must live in a separate table (similar to how we do domains today). And the rest of the code needs to change to store both id/name. Finally, there is backwards compatibility with cassandra.  This is a much bigger discussion / change - open to this - but belongs in a different diff.

Reply:
Makes sense.  Let's create an issue to track this.